test/integration/developer_can_logout_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class DeveloperCanLogoutTest < ActionDispatch::IntegrationTest
+  test "contractor can logout" do
+    developer = create(:developer)
+    visit login_path
+
+    within '#developer-login-form' do
+      fill_in "session[email]", with: developer.email
+      fill_in "session[password]", with: developer.password
+      click_on "Developer Login"
+    end
+
+    assert page.has_content?("Logged in: Developer #{developer.name}")
+
+    click_on "Logout"
+
+    assert_equal login_path, current_path
+    assert page.has_content?("Logged out #{developer.name}")
+    refute page.has_content?("Logged in: Developer #{developer.name}")
+
+    visit '/developer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.